Add Docker resource requirements to dev container setup docs

ruokun-niu opened the pull request:
This PR adds recommended Docker resource allocation section to dev container documentation for our tutorials. Currently, a section was added to the building comfort, risy container, absense of change and getting start tutorials. I was able to go through these tutorials smoothly with this config. It would be great if people could do some testing with these resource configs to see if things work well.

I did not add a section for the curbside pickup tutorial yet, as I know that Aman is working on it at the moment. I'll add that section once this tutorial is updated

agentofreality commented:
The additional information in these additions is useful, but it leaves a bad/confusing flow for the user. Prior to the additions, the instructions told the user the steps they needed to get the dev container running. I think the issue is the large heading that makes it look like the rest of the content is about the recommended settings; the recommended settings now strand the final step about loading the dev container at the bottom.

<img width="731" alt="image" src="https://github.com/user-attachments/assets/eb662d6e-b74f-4656-854b-c3230c3485a9" />

consider moving the last para up above the recommended settings and a smaller heading.

danielgerlag commented on docs/content/getting-started/_index.md:
```diff
@@ -82,10 +82,26 @@ To use the Drasi Getting Started Dev Container, you will need to install:
+- **CPU**: 3 cores or more
+- **Memory**: 4 GB or more
+- **Swap**: 1 GB or more
+- **Disk**: 272 GB available space 
```
272 GB of disk space seems excessive? Mine was working with 50GB.

ruokun-niu replied:
I will try to run the tutorials again with 50 GB

ruokun-niu commented:
@agentofreality @danielgerlag I have updated the recommendation for disk space and moved the recommendation section to a lower place. Here is an example:
![Screenshot 2025-06-20 at 3 06 27 PM](https://github.com/user-attachments/assets/b0d5fec2-28a7-4354-8668-664db9806d4f)

amansinghoriginal commented on docs/content/tutorials/building-comfort/_index.md:
```diff
@@ -264,6 +264,23 @@ Once the solution is open in VS Code, follow these steps:
 - Select `Dev Containers: Rebuild and Reopen in Container`.
 - Select the `Building Comfort with Drasi` option to launch this tutorial.
 
+##### Recommended Docker Resources
```
Can we add similar section for curbside one as well?
